### PR TITLE
update dependencies and node version for Travis

### PR DESCRIPTION
**Pull request by nebulon42**
Updating some dependencies that were reported as outdated.
Updating node versions for Travis, since older versions are now EOL (see https://github.com/nodejs/LTS for details).

### CODE REVIEW

**Comment by nebulon42** [comment]
Travis shows that node-mapnik is not (yet) running on Node 7. I'm slightly tending towards leaving this build in anyway.

**Comment by yohanboniface** [comment]
I'd rather remove node 7 for now: having Travis badge green is a good helper. If we expect it to be red, we may miss the real reason. What do you think?

**Comment by pnorman** [comment]
Shouldn't merge something that's not passing travis. Either remove node 7 or set it so it's not require to pass.

**Comment by nebulon42** [comment]
I have removed Node 7.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - "5.5"
-  - "4.2"
-  - "0.10"
```
Comment (pnorman) [inline .travis.yml]: Does this mean we're dropping support on Ubuntu trusty?

Comment (nebulon42) [inline .travis.yml]: Not necessarily. It is still possible and not hard to install a more recent node.js version on Trusty by following https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions. The readme also mentions that you should run the latest LTS release.

I have no problem in leaving in older versions, but as these are EOL I'm reluctant to do anything if they start failing.

**Pull request merged**